aten/src/ATen/native/native_functions.yaml

```diff
@@ -8142,6 +8142,16 @@
     CPU: searchsorted_cpu
     CUDA: searchsorted_cuda
 
+- func: _convert_coo_to_csr.Tensor(Tensor self, int size, *, bool out_int32=False) -> Tensor
```

Comment:
I think the name of the function should contain the "indices" word because this function does not convert COO -> CSR tensor, but converts the row indices.